src/lib/app-provider/queries/simulateFee.ts

```diff
@@ -66,7 +66,7 @@ export const useSimulateFeeQuery = ({
     queryFn: async ({ queryKey }) => simulateFn(queryKey[3] as ComposedMsg[]),
     enabled,
     keepPreviousData: true,
-    retry: false,
+    retry: 2,
```

Comment:
Do we have to change for other simulate fee?

Reply:
its not wrapped in useQuery